Make my recent works

fuadmoin opened the pull request:
'my recent works' has been added to the portfolio.html and styled with CSS.

fuadmoin committed:
Modify portfolio.html

my recent works has been added to the portfolio.html and styled with css.

fuadmoin committed:
Remove duplicate files

fuadmoin committed:
Add missing file

fuadmoin committed:
Remove duplicate files

fuadmoin committed:
Edit style.css

fuadmoin committed:
Edit style.css

KanzaTahreem reviewed (commented):
# INVALID ❌ 

Hi @fuadmoin  👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😓

## Invalid Code Review Request ❌
- [ ] Unfortunately, you have submitted a project that is not professional and lacks a few `headers/sections` according [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) Please make sure that **all headers/sections are present, but the content should be customized to this project repository.**

Please ensure that you professionally document the README file to reflect the project
You can take a step back and read the [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more clarity 💡

_**Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready**_ 🤝🙏

Cheers and Happy coding!👏👨‍💻💻

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

fuadmoin committed:
Modify readme.md

roshan-bajgain commented on porfolio.html:
```diff
@@ -50,6 +50,199 @@ <h1 class="primary-text">Hey There. <br>
+                        <p class="cwo-supporting-text">A daily selection of privately <br>
+                            personalized reads; no accounts or <br>
+                            sign-ups required. has been the <br>
+                            industry's standard</p>
+                        <ul class="cwo-tags">
+                            <li class="cw-li cw-html"><a href="" ><small class="cw-text">html</small></a></li>
+                            <li class="cw-li cw-bootstrap"><a href="" ><small class="cw-text">bootstrap</small></a></li>
+                            <li class="cw-li cw-ruby"><a href="" ><small class="cw-text">Ruby</small></a></li>
+                        </ul>
+                        
+                    </article>  
+                    <div class="mask-group"></div>
+
+
+                </section>
+                <section class=" cardworks cardwork-six">
+                    <div class="action">
+                        
+                        <button type="button" disabled class="btn"><small class="btntxt">See Project</small></button>
+                        <hr class="rectangle">
+                    </div>
+                    <article class="cardwork-one-rb">
+                        <p class="cwo-title-post">Profesional Art <br>Printing Data</p>
+                        <p class="cwo-supporting-text">A daily selection of privately <br>
+                            personalized reads; no accounts or <br>
+                            sign-ups required. has been the <br>
+                            industry's standard</p>
+                        <ul class="cwo-tags">
+                            <li class="cw-li cw-html"><a href="" ><small class="cw-text">html</small></a></li>
+                            <li class="cw-li cw-bootstrap"><a href="" ><small class="cw-text">bootstrap</small></a></li>
+                            <li class="cw-li cw-ruby"><a href="" ><small class="cw-text">Ruby</small></a></li>
+                        </ul>
+                        
+                    </article>  
+                    <div class="mask-group"></div>
+
+
+                </section>
+            </div>
+
```
- [ ] It would be better if you remove the `div` tag as you already cover the content with the `section` tag you can use the div tag inside the section for better semantic HTML.

fuadmoin replied:
@roshan-bajgain  the div is a container for the cards. I have 6 cards (from <section class=" cardworks cardwork-one"> till <section class=" cardworks cardwork-six">) and I contained them with div class. So I can't remove it. And I used CSS grid property on this div (to put the cards in order).

roshan-bajgain commented on porfolio.html:
```diff
@@ -50,6 +50,199 @@ <h1 class="primary-text">Hey There. <br>
+                            personalized reads; no accounts or <br>
+                            sign-ups required. has been the <br>
+                            industry's standard</p>
+                        <ul class="cwo-tags">
+                            <li class="cw-li cw-html"><a href="" ><small class="cw-text">html</small></a></li>
+                            <li class="cw-li cw-bootstrap"><a href="" ><small class="cw-text">bootstrap</small></a></li>
+                            <li class="cw-li cw-ruby"><a href="" ><small class="cw-text">Ruby</small></a></li>
+                        </ul>
```
- [ ] Please stick to the design as much as possible. please remove the underline from `li` tag so it matches the design
### Your design
![his](https://user-images.githubusercontent.com/81727988/213196767-c1e30eda-73aa-42c1-9423-a6c29a776cf0.jpg)

roshan-bajgain commented on porfolio.html:
```diff
@@ -50,6 +50,199 @@ <h1 class="primary-text">Hey There. <br>
+                            <li class="cw-li cw-html"><a href="" ><small class="cw-text">html</small></a></li>
+                            <li class="cw-li cw-bootstrap"><a href="" ><small class="cw-text">bootstrap</small></a></li>
+                            <li class="cw-li cw-ruby"><a href="" ><small class="cw-text">Ruby</small></a></li>
+                        </ul>
```
- [ ] Please stick to the design as much as possible. please remove the underline from `li` tag so it matches the design
### Your design
![his](https://user-images.githubusercontent.com/81727988/213196767-c1e30eda-73aa-42c1-9423-a6c29a776cf0.jpg)

roshan-bajgain commented on porfolio.html:
```diff
@@ -50,6 +50,199 @@ <h1 class="primary-text">Hey There. <br>
+                            <li class="cw-li cw-html"><a href="" ><small class="cw-text">html</small></a></li>
+                            <li class="cw-li cw-bootstrap"><a href="" ><small class="cw-text">bootstrap</small></a></li>
+                            <li class="cw-li cw-ruby"><a href="" ><small class="cw-text">Ruby</small></a></li>
+                        </ul>
```
- [ ] Please stick to the design as much as possible. please remove the underline from `li` tag so it matches the design
### Your design
![his](https://user-images.githubusercontent.com/81727988/213196767-c1e30eda-73aa-42c1-9423-a6c29a776cf0.jpg)

roshan-bajgain commented on porfolio.html:
```diff
@@ -50,6 +50,199 @@ <h1 class="primary-text">Hey There. <br>
+                            <li class="cw-li cw-html"><a href="" ><small class="cw-text">html</small></a></li>
+                            <li class="cw-li cw-bootstrap"><a href="" ><small class="cw-text">bootstrap</small></a></li>
+                            <li class="cw-li cw-ruby"><a href="" ><small class="cw-text">Ruby</small></a></li>
+                        </ul>
```
- [ ] Please stick to the design as much as possible. please remove the underline from `li` tag so it matches the design
### Your design
![his](https://user-images.githubusercontent.com/81727988/213196767-c1e30eda-73aa-42c1-9423-a6c29a776cf0.jpg)

roshan-bajgain commented on porfolio.html:
```diff
@@ -50,6 +50,199 @@ <h1 class="primary-text">Hey There. <br>
+                            <li class="cw-li cw-html"><a href="" ><small class="cw-text">html</small></a></li>
+                            <li class="cw-li cw-bootstrap"><a href="" ><small class="cw-text">bootstrap</small></a></li>
+                            <li class="cw-li cw-ruby"><a href="" ><small class="cw-text">Ruby</small></a></li>
+                        </ul>
```
- [ ] Please stick to the design as much as possible. please remove the underline from `li` tag so it matches the design
### Your design
![his](https://user-images.githubusercontent.com/81727988/213196767-c1e30eda-73aa-42c1-9423-a6c29a776cf0.jpg)

roshan-bajgain commented on porfolio.html:
```diff
@@ -50,6 +50,199 @@ <h1 class="primary-text">Hey There. <br>
+                            <li class="cw-li cw-html"><a href="" ><small class="cw-text">html</small></a></li>
+                            <li class="cw-li cw-bootstrap"><a href="" ><small class="cw-text">bootstrap</small></a></li>
+                            <li class="cw-li cw-ruby"><a href="" ><small class="cw-text">Ruby</small></a></li>
+                        </ul>
```
- [ ] Please stick to the design as much as possible. please remove the underline from `li` tag so it matches the design
### Your design
![his](https://user-images.githubusercontent.com/81727988/213196767-c1e30eda-73aa-42c1-9423-a6c29a776cf0.jpg)

roshan-bajgain commented on porfolio.html:
```diff
@@ -50,6 +50,199 @@ <h1 class="primary-text">Hey There. <br>
+                        <li class="html"><a href="" ><small class="htmlt">html</small></a></li>
+                        <li class="bootstrap"><a href="" ><small class="bootstrapt">bootstrap</small></a></li>
+                        <li class="ruby"><a href="" ><small class="rubyt">Ruby</small></a></li>
+                    </ul>
+                    <button type="button" disabled class="see-project"><small class="see-projectt">See Project</small></button>
+
```
- [ ] Good job so far but there is a small change you need to make before going to the next project. I would suggest you please use import the google font and its uses `font-family: 'Crete Round';` for hx tag and for a paragraph it uses `font-family: 'Inter';` Please see the font and use it. you use the font but it not implementing as you did not import or use it in an HTML file.
- [ ] I would suggest you please fix the spacing as you need to stick to the Figma design as much as possible.
- [ ] I would suggest please work on the `li` tag as it is different from then design. Please see below for reference.
- [ ] I would suggest please make your heading as in Figma design by using the same font.
### Your design
![your](https://user-images.githubusercontent.com/81727988/213195732-d4d5a753-ff12-44af-b7a4-59b9fe1967b3.jpg)


### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/213195764-b31d5842-5b99-43d6-b6e9-65a1b299af9f.jpg)

fuadmoin replied:
@roshan-bajgain  thank you for the review.  font-family: 'Crete Round'; and font-family: 'Inter';  are giving me a stylelint linter error, that's why I changed them.

roshan-bajgain commented on porfolio.html:
```diff
@@ -50,6 +50,199 @@ <h1 class="primary-text">Hey There. <br>
+                    <div class="action">
+                        
+                        <button type="button" disabled class="btn"><small class="btntxt">See Project</small></button>
+                        <hr class="rectangle">
```
- [ ] According to the requirement the buttons should follow the button styling: Active, Hover and Pressed (see design). Please implement this for all buttons.
### Design
![fimga](https://user-images.githubusercontent.com/81727988/213199148-5347a467-bb87-43ae-9935-263ba903d405.jpg)

roshan-bajgain reviewed (changes requested):
Hi @fuadmoin, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] As the project requirement you should implement the "About myself" section but I notice you missed this part. Please do it and make sure that you stick to the design as much as possible as the next reviewer will review this `About section` and make sure that everything is good to go. 😃 
### Design
![fimga](https://user-images.githubusercontent.com/81727988/213198159-f2dedbe0-6a4c-490f-a94a-fddc96bab598.jpg)
- [ ] I cannot comment there so I am leaving here your main container of the projects has to be display:grid. Please implement it.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

fuadmoin committed:
Modify portfolio.html

 I added hover effect and fixed some alignment issues

fuadmoin committed:
Add about me section

I added about me section and in the portfolio.html and styled it with css in style.css

Tufahel commented on porfolio.html:
```diff
@@ -50,6 +50,245 @@ <h1 class="primary-text">Hey There. <br>
+               <img src="./images/yoga.png" alt="a place holder image" class="yoga"> 
+                <div class="right-block">
+                    <h3 class="title-post">Multi-Post Stories</h3>
+                    <p class="supporting-text-rw">
+                        A daily selection of privately personalized <br>
+                        reads;  no accounts or sign-ups required. <br>
+                         has been the industry's 
+                        standard dummy<br> text ever since the 1500s,
+                         when an <br>unknown printer took a standard dummy<br>text.
+                    </p>
+                    <ul class="tags">
+                        <li class="css"><small class="csst">css</small></li>
+                        <li class="html"><small class="htmlt">html</small></li>
+                        <li class="bootstrap"><small class="bootstrapt">bootstrap</small></li>
+                        <li class="ruby"><small class="rubyt">Ruby</small></li>
+                    </ul>
+                    <button type="button"  class="see-project"><small class="see-projectt">See Project</small></button>
```
- [ ] `Gaps` between `paragraph and ul` and `gaps` between `ul and button` should same as per the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27&t=4VlBei1WFE8FxHjg-0) design and previous reviewer suggestion. Kindly fix this issue.

***Your Design***
![Screenshot from 2023-01-20 02-32-12](https://user-images.githubusercontent.com/60083437/213554205-0628541d-6632-439d-a715-bda3bbb10cd9.png)

***Figma Design***
![Screenshot from 2023-01-20 02-36-45](https://user-images.githubusercontent.com/60083437/213554358-74c0345a-349e-4f9e-b068-9880916cbc22.png)

fuadmoin replied:
@Tufahel well on the Figma you are actually measuring the space between the container and the tags, that's why it seemed off. Please check it again. I am not saying it is perfect but the comparison is not fair.

Tufahel commented on style.css:
```diff
@@ -238,11 +235,932 @@ body {
+  width: 327px;
+  height: 48px;
+  background: #ff6b00;
+}
```
- [ ] You have implemented `button interaction` as per the previous reviewer's suggestion. Kindly fix the border of the button by adding `border: 0` in the `rectangle` class CSS so that it matches the Figma Design. In your design, there is a border of `gray` color.

***Your Design***
![Screenshot from 2023-01-20 02-48-06](https://user-images.githubusercontent.com/60083437/213556343-3509c34a-36de-48cb-99fe-853cd2a2471a.png)

Tufahel commented on style.css:
```diff
@@ -238,11 +235,932 @@ body {
+  top: 370px;
+  border-radius: 0;
+  position: absolute;
+}
```
- [ ] Kindly fix the button's border by adding `border: 0` in the `btn-about` class CSS also, so that it matches the Figma Design. In your design, there is a border of `gray` color.

***Your Design***
![Screenshot from 2023-01-20 02-49-48](https://user-images.githubusercontent.com/60083437/213557105-a91e8794-0b7f-460b-a97f-2e6d6aa12fd2.png)

Tufahel reviewed (changes requested):
# Status: Required Changes :recycle:
Hi @fuadmoin :wave: 

Well done on implementing some of the required changes. :+1:  
You need to fix some issues before moving to the next project. :running_man: 

## Highlights 🌟

![Alt Text](https://c.tenor.com/vOLW_UQMySIAAAAC/almost-there.gif)

✔️ Linters checks are passing
✔️ Use GitHub Flow
✔️ Meaningful commit messages.

## Required Changes ♻️

_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

fuadmoin committed:
Modify style.css

levy002 reviewed (approved):
## Hello @team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
